Some of MDN community docs translations (#7792)
mdn/translated-content: files/zh-cn/mdn/community/contributing/getting_started/index.md

---
{{MDNSidebar}}

我们是一个开放包容的开发者社区，目标是为更好的互联网提供资源，无关品牌、浏览器或平台。任何人都可以贡献自己的力量，你的每一份贡献都会让我们更强大。与我们一起，持续推动互联网技术向更伟大的方向革新——就在这里，与你一起。

[加入我们吧！](https://github.com/mdn/mdn-community/)

## 我能做些什么？

> 想要为 MDN 做贡献的道路很多，取决于你的技能和兴趣。对于每一种任务，我们会给出简要的解释和预计完成的大致时间。

Reviewer: ```suggestion
可以为 MDN 做贡献的方式很多，取决于你的技能和兴趣。对于每一种任务，我们会给出简要的解释和预计完成的大致时间。
```

> 如果你不知道该做些什么，欢迎向我们[寻求帮助](https://github.com/mdn/mdn-community/)。

> > 也需要注意，我们麻雀虽小，但五脏俱全的团队正在维护着这个存储库，为了保护带宽，将关闭无关的讨论。

Reviewer: 17 行俗语拆开翻译有点奇怪，需要商榷。

Author: > 17 行俗语拆开翻译有点奇怪，需要商榷。

这里的原文是 `our small, but mighty docs team`，直译的行文或许对中文来说，有点矛盾？“渺小但强大的文档团队”？感觉我也想不出接近的译法……

Author: > “保护宽带”一词不应直译

想了想，大概还是应该往“流量”那方面意思走走

```suggestion
> 也需要注意，我们麻雀虽小，但五脏俱全的团队正在维护着这个存储库，为了节省流量，将关闭无关的讨论。
```

Reviewer: > > 17 行俗语拆开翻译有点奇怪，需要商榷。
> 
> 这里的原文是 `our small, but mighty docs team`，直译的行文或许对中文来说，有点矛盾？“渺小但强大的文档团队”？感觉我也想不出接近的译法……

从意思表达上来说，可以译成我们的团队“麻雀虽小五脏俱全”，但是这似乎不符合技术文档的风格。

Reviewer: 不对，看了一下原文，这里的意思不是突出团队的强大，而是想说”我们这有人维护，别捣乱“的意思。
```suggestion
> 也需要注意，我们的团队人数虽少，但是都十分专业。为了节省流量和维护成本，我们将关闭无关的讨论。
```

Author: 啊，这样就明白了 😄

Author: > 不对，看了一下原文，这里的意思不是突出团队的强大，而是想说”我们这有人维护，别捣乱“的意思。

原来如此 😄

Reviewer 2: For the meaning of `bandwidth`, I think it should mean:

> the energy or mental capacity required to deal with a situation.

![image](https://user-images.githubusercontent.com/15844309/185821195-118ff2ad-7b63-437b-9efb-40407f6f7106.png)

So `精力` is appropriate

```suggestion
> 也需要注意，我们的文档维护团队人数虽少，但是都十分专业。为了避免分散精力，我们将关闭无关的讨论。
```

> > 如果你不知道该做些什么，欢迎向我们[寻求帮助](https://github.com/mdn/mdn-community/)。
> > 也需要注意，我们麻雀虽小，但五脏俱全的团队正在维护着这个存储库，为了保护带宽，将关闭无关的讨论。

Reviewer 2: ```suggestion
> 如果你不知道该做些什么，欢迎向我们[寻求帮助](https://github.com/mdn/mdn-community/)。也需要注意，我们的文档维护团队人数虽少，但是都十分专业。为了避免分散精力，我们将关闭无关的讨论。```

我们创建了一个[贡献者任务板](https://github.com/orgs/mdn/projects/25/views/1)来帮助你寻找为项目做出重大影响的贡献机会，任务板包含有概述部分和特定贡献类型的分离视图。

### 我准备好了！

为了做出贡献，你需要拥有一个 GitHub 账号，如果没有，你可以[创建一个](https://github.com/signup)。对于 GitHub 新手，我们鼓励你学习以下由 GitHub 提供的免费课程和阅读资料。当充分补充了这方面的知识后，你可以专心做贡献，而不必同时学习新的技能。

> 注意：没必要为完成 _所有_ 的课程任务和阅读资料不知所措。在了解了“GitHub 简介”课程后，你应该拥有了足够的知识来为我们贡献。

- [GitHub 简介](https://github.com/skills/introduction-to-github)
- [安装并设定 Git](https://docs.github.com/cn/get-started/quickstart/set-up-git)
- [GitHub 工作流](https://docs.github.com/cn/get-started/quickstart/github-flow)
- [使用 Markdown](https://github.com/skills/communicate-using-markdown)

### 额外的阅读学习资料

- [开源项目基本礼节](/zh-CN/docs/MDN/Community/Open_source_etiquette)：如果你还没有为开源项目贡献过，鼓励你首先阅读这篇文档。
- [学习 Web 开发](/zh-CN/docs/Learn)：如果你是 HTML、CSS 或 JavaScript 新手，我们为你准备了丰富的内容帮你了解这些知识。
- [协作处理拉取请求](https://docs.github.com/cn/pull-requests/collaborating-with-pull-requests)

某些写作相关的机会需要对英语的足够了解。即使你的语法不够好，也不用担心！我们的团队会保证 MDN 上的内容足够好。也就是说，有人会在一旁确保工作的整洁和行文的正确性。

一旦你决定了你所做的任务种类，就可以前往[贡献者任务板](https://github.com/orgs/mdn/projects/25/views/1)中挑选议题，并在评论区中留下评论和提及 `@mdn/mdn-community-engagement` 团队。我们队伍中的成员会回应，且将议题分配给你。

这保证两个人的工作不会“撞车”，并且可以知道在工作遇到困难的时候，你该联系的人。

### 贡献

> 当贡献时，视为你同意 [Attribution-ShareAlike license](https://creativecommons.org/licenses/by-sa/4.0/)（或在所编辑页面中另行指定的）许可。另外，代码示例的许可为 [Creative Commons CC-0](https://creativecommons.org/share-your-work/public-domain/cc0/)（一种公有领域贡献的协议）。

Author: ```suggestion
当贡献时，视为你同意 [Attribution-ShareAlike license](https://creativecommons.org/licenses/by-sa/4.0/)（或在所编辑页面中已经指定的其它版权）许可。另外，代码示例的许可为 [Creative Commons CC-0](https://creativecommons.org/share-your-work/public-domain/cc0/)（专用于公有领域）。
```

> 当贡献时，视为你同意 [Attribution-ShareAlike license](https://creativecommons.org/licenses/by-sa/4.0/)（或在所编辑页面中另行指定的）许可。另外，代码示例的许可为 [Creative Commons CC-0](https://creativecommons.org/share-your-work/public-domain/cc0/)（一种公有领域贡献的协议）。

Author: 沿用了那个页面的说法
